File: 프로그래머스/unrated/181912. 배열 만들기 5/배열 만들기 5.java
import java.util.*;

class Solution {
    public int[] solution(String[] intStrs, int k, int s, int l) {
        
        ArrayList<Integer> list = new ArrayList<>();
        for (int i = 0; i < intStrs.length; i++) {
            String ss = intStrs[i].substring(s, s + l);
            if (Integer.parseInt(ss) > k) {
                list.add(Integer.parseInt(ss));
            }
        }
        
        int[] answer = new int[list.size()];
        for (int i = 0; i < answer.length; i++) {
            answer[i] = list.get(i);
        }
        
        return answer;
    }
}

File: 프로그래머스/unrated/181912. 배열 만들기 5/배열 만들기 5.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<string> intStrs, int k, int s, int l) {
    vector<int> answer;
    for (int i = 0; i < intStrs.size(); i++) {
        string ss = intStrs[i].substr(s, l);
        int a = stoi(ss);
        if (a > k) {
            answer.push_back(a);
        }
    }

    return answer;
}